proposals/advisory-db.md

```diff
@@ -146,6 +148,8 @@ To begin with, the HF executive team will assemble a group of five volunteers, w
 
 We expect that the work done by this team will occur mostly asynchronously, but we plan to have meetings a few times per year in order to have discussions about how the group is working and how it can be improved.
 
+Because CVSS is a bit complex, PRs without CVSS will be provisionally accepted, and the volunteers will work with reporters to help them fill out the field appropriately.
```

Comment:
Might be worth linking this here - https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator